### DISCUSSION TITLE
Placing on wall requires placing on ground first

### DISCUSSION THREAD

**Discussion by bjarque** [google/model-viewer#1952] [Q&A]
The new ar-placement tag doesn't work really well with adding items to a wall in iOS. I have to add it to floor first, when the ios webxr viewer asks for moving the phone around for initial placement. After that it is somewhat possible to drag it onto a wall, but it requires that the wall is total clean, and and it cant be placed on the wall if theres items on it.

Actually this was also the behaviour before the ar-placement tag was added, so it doens't really seem to add any functionality, at least not on iOS.

### Browser Affected

- [ ] Chrome
- [x] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [ ] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment]
That is correct, we have no control over QuickLook on iOS. This attribute only affects WebXR and SceneViewer.
